### PR TITLE
Ensure Cascade Deletion with Ownreference

### PR DESCRIPTION
**Pull request by hangscer8**
#### What type of PR is this?

/kind bug

#### Which issue(s) this PR fixes:

Fixes #

### CODE REVIEW

**Comment by hangscer8** [comment]
test:

`before`:
企业微信截图_479ed2fb-9771-4eb5-99c9-96d1f0e1fdf3

`after deleting cr "cluster" instance`:
企业微信截图_fb714985-f9bd-41bd-a6ff-c866924ba68a

File: pkg/controllers/clusterops/controller.go [hunk 1]
```diff
@@ -846,6 +849,18 @@ func (c *Controller) CopySecret(clusterOps *clusteroperationv1alpha1.ClusterOper
 	return newSecret, nil
 }
 
+func (c *Controller) UpdateOwnReferenceToClusterOps(clusterOps *clusteroperationv1alpha1.ClusterOperation) error {
+	if err := util.UpdateOwnReference(
```
Comment (tu1h) [inline pkg/controllers/clusterops/controller.go:853]:
```suggestion
	return util.UpdateOwnReference(
		c.ClientSet,
		clusterOps.Spec.ConfigDataList(),
		clusterOps.Spec.SecretDataList(),
		*metav1.NewControllerRef(clusterOps, clusteroperationv1alpha1.SchemeGroupVersion.WithKind("ClusterOperation")),
	)
```
here just return

**Pull request merged**